app/scripts/setup.sh

```diff
@@ -74,18 +74,24 @@ function createSymlinks() {
     esac
 }
 
-function checkoutSilme() {
-    # Check out SILME library to a specific version (0.8.0)
-    if [ ! -d $libraries/silme/.hg ]
+function setupExternalLibraries() {
+    # Check out or update compare-locales library
+    version="RELEASE_1_2_1"
```

Comment:
I'm not sure storing it in config would be a good solution, mostly because we wouldn't have clear control over it (obsolete config -> obsolete library version, also potentially non existent version).

I don't expect compare-locales to be updated frequently, I just want to make sure we have the version we want (e.g. 1.2 had bugs, I want to make sure we have 1.2.1). The cost in terms of time is acceptable, since it's run only in setup.sh

If compare-locales changes significantly, we would still need an update to the main Python script.